koa-crud/src/graphql/type-defs/members.ts

```diff
@@ -12,6 +12,17 @@ export default gql`
     deleteMember(id: ID!): Boolean!
   }
 
+  type MemberConnectionEdge {
+    node: Member
+    cursor: String
+  }
+
+  type MemberConnection {
+    totalCount: Int
```

Comment:
`totalCount` should be required. if there is no data then return zero (0)

Reply:
noted sir